debian/libseafile0.symbols

```diff
@@ -0,0 +1,430 @@
+ seafile_sync_task_get_state@Base 5.1.2
+ seafile_sync_task_get_tx_id@Base 5.1.2
+ seafile_sync_task_get_type@Base 5.1.2
+ seafile_sync_task_new@Base 5.1.2
+ seafile_sync_task_set_dest_id@Base 5.1.2
+ seafile_sync_task_set_error@Base 5.1.2
+ seafile_sync_task_set_force_upload@Base 5.1.2
+ seafile_sync_task_set_is_sync_lan@Base 5.1.2
+ seafile_sync_task_set_repo_id@Base 5.1.2
+ seafile_sync_task_set_state@Base 5.1.2
+ seafile_sync_task_set_tx_id@Base 5.1.2
+ seafile_task_construct@Base 5.1.2
+ seafile_task_get_block_done@Base 5.1.2
+ seafile_task_get_block_total@Base 5.1.2
+ seafile_task_get_dest_id@Base 5.1.2
+ seafile_task_get_dsize@Base 5.1.2
+ seafile_task_get_error_str@Base 5.1.2
+ seafile_task_get_from_branch@Base 5.1.2
+ seafile_task_get_fs_objects_done@Base 5.1.2
+ seafile_task_get_fs_objects_total@Base 5.1.2
+ seafile_task_get_rate@Base 5.1.2
+ seafile_task_get_repo_id@Base 5.1.2
+ seafile_task_get_rsize@Base 5.1.2
+ seafile_task_get_rt_state@Base 5.1.2
+ seafile_task_get_state@Base 5.1.2
+ seafile_task_get_to_branch@Base 5.1.2
+ seafile_task_get_ttype@Base 5.1.2
+ seafile_task_get_tx_id@Base 5.1.2
+ seafile_task_get_type@Base 5.1.2
+ seafile_task_new@Base 5.1.2
+ seafile_task_set_block_done@Base 5.1.2
+ seafile_task_set_block_total@Base 5.1.2
+ seafile_task_set_dest_id@Base 5.1.2
+ seafile_task_set_dsize@Base 5.1.2
+ seafile_task_set_error_str@Base 5.1.2
+ seafile_task_set_from_branch@Base 5.1.2
+ seafile_task_set_fs_objects_done@Base 5.1.2
+ seafile_task_set_fs_objects_total@Base 5.1.2
+ seafile_task_set_rate@Base 5.1.2
+ seafile_task_set_repo_id@Base 5.1.2
+ seafile_task_set_rsize@Base 5.1.2
+ seafile_task_set_rt_state@Base 5.1.2
+ seafile_task_set_state@Base 5.1.2
+ seafile_task_set_to_branch@Base 5.1.2
+ seafile_task_set_ttype@Base 5.1.2
+ seafile_task_set_tx_id@Base 5.1.2
+ seafile_trash_repo_construct@Base 5.1.2
+ seafile_trash_repo_get_del_time@Base 5.1.2
+ seafile_trash_repo_get_head_id@Base 5.1.2
+ seafile_trash_repo_get_owner_id@Base 5.1.2
+ seafile_trash_repo_get_repo_id@Base 5.1.2
+ seafile_trash_repo_get_repo_name@Base 5.1.2
+ seafile_trash_repo_get_size@Base 5.1.2
+ seafile_trash_repo_get_type@Base 5.1.2
+ seafile_trash_repo_new@Base 5.1.2
+ seafile_trash_repo_set_del_time@Base 5.1.2
+ seafile_trash_repo_set_head_id@Base 5.1.2
+ seafile_trash_repo_set_owner_id@Base 5.1.2
+ seafile_trash_repo_set_repo_id@Base 5.1.2
+ seafile_trash_repo_set_repo_name@Base 5.1.2
+ seafile_trash_repo_set_size@Base 5.1.2
+ seafile_web_access_construct@Base 5.1.2
+ seafile_web_access_get_obj_id@Base 5.1.2
+ seafile_web_access_get_op@Base 5.1.2
+ seafile_web_access_get_repo_id@Base 5.1.2
+ seafile_web_access_get_type@Base 5.1.2
+ seafile_web_access_get_username@Base 5.1.2
+ seafile_web_access_new@Base 5.1.2
+ seafile_web_access_set_obj_id@Base 5.1.2
+ seafile_web_access_set_op@Base 5.1.2
+ seafile_web_access_set_repo_id@Base 5.1.2
+ seafile_web_access_set_username@Base 5.1.2
+ seafile_web_query_access_token@Base 5.1.2
```

Comment:
Could we generate the symbols file at build time instead of keeping this static list in the source tree?